### PR TITLE
Use package_endpoint parameter

### LINKED ISSUE #40: Parameterize service URL in feed actions
Currently, the feed actions have a hardcoded URL for the feed service provider. This should be parameterized so that this URL can be set during deployment.

### PR DESCRIPTION
**Pull request by bjustin-ibm**
This parameter should be set to the correct value as a bound parameter when the feed action is deployed.

Resolves #40

### CODE REVIEW

File: action/feedAction.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function main(params) {
     var namespace = encodeURIComponent(triggerComponents[1]);
     var trigger = encodeURIComponent(triggerComponents[2]);
 
-    var feedServiceURL = 'http://owkafkafeedprovider.mybluemix.net/triggers/' + namespace + '/' + trigger;
+    var feedServiceURL = 'http://' + params.package_endpoint + '/triggers/' + namespace + '/' + trigger;
```
Comment (dubeejw) [inline action/feedAction.js]: Add a check for the package_endpoint. Should be good after that.

**Comment by bjustin-ibm** [comment]
This will need to be reworked due to #33

**Comment by bjustin-ibm** [comment]
@dubeejw Please have one more look now that I've applied it to the two new actions.

File: action/kafkaFeed.js [hunk 2]
```diff
@@ -8,11 +8,16 @@ var request = require('request');
  *  @param {string} endpoint - address to OpenWhisk deployment
  */
 function main(params) {
+    if(!params.package_endpoint) {
```
Comment (dubeejw) [inline action/kafkaFeed.js]: LGTM

**Pull request merged**